scripts/kata-arch.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+#
+# Copyright (c) 2017-2018 Intel Corporation
```

Comment:
2017-2018 - did we clone this from or start writing this last year? (just curious)

Reply:
Fixed.